CMakeLists.txt

```diff
@@ -37,3 +37,8 @@ set_target_properties(${MODULE_NAME} PROPERTIES PREFIX "")
 
 install(TARGETS ${MODULE_NAME} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${NAMESPACE}/OCDM)
 
+if(NOT "${CMAKE_FIND_ROOT_PATH}" STREQUAL "")
+    install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink /etc/playready  ${CMAKE_CURRENT_BINARY_DIR}/playready)")
+    install(DIRECTORY DESTINATION ${CMAKE_INSTALL_PREFIX}/../root/OCDM)
```

Comment:
The /.../root needs to be replaced by the ${PERSISTENT_PATH} variable.

Reply:
Handling that already on sander/new-ocdm-interface